src/engraving/rendering/score/systemlayout.cpp

```diff
@@ -1417,10 +1417,14 @@ void SystemLayout::layoutSystemElements(System* system, LayoutContext& ctx)
             }
 
             if (s->isType(SegmentType::TimeSigType)) {
-                TimeSig* ts = toTimeSig(s->element(e->track()));
-                TimeSigPlacement timeSigPlacement = ts->style().styleV(Sid::timeSigPlacement).value<TimeSigPlacement>();
+                EngravingItem* el = s->element(e->track());
+                TimeSig* timeSig = el ? toTimeSig(el) : nullptr;
```

Comment:
FWIW, I believe calling `toTimeSig` with `nullptr` is okay, since it does a null check internally. That would allow for a slight simplification. But it's not important.